adafruit_as7341.py

```diff
@@ -640,30 +656,30 @@ def _set_smux(self, smux_addr, smux_out1, smux_out2):
         self._write_register(smux_addr, smux_byte)
```

Comment:
I for the life of my can't figure out how to place this review higher up at the function definition here, sorry, still getting used to reviewing!  It looks like based on the way `SMUX_IN` and `SMUX_OUT` were set up as `CVs` to dynamically add values, the definition should be:

`def _set_smux(self, smux_addr: int, smux_out1: int, smux_out2: int) -> None:`

That's basically how the libraries use the CV like structure, to dynamically create enum-like values, so here we type it with the same typing as the `CV`'s `value` in that tuple!

Reply:
Got it - thanks! Adjusting.